src/Data/Container/GreatestFixpoint.agda

```diff
@@ -0,0 +1,23 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Greatest fixpoint for containers
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K --guardedness #-}
+
+module Data.Container.GreatestFixpoint where
+
+open import Level using (Level; _⊔_)
+open import Codata.Musical.M using (M)
+open import Data.Container using (Container)
+
+private
+  variable
+    s p : Level
+
+-- This lives in its own module due to its unsafe use of sized types.
```

Comment:
This comment is wrong as well, my fault, sorry!

Reply:
It used to be correct but then I moved this greatest fixpoint from sizes to guardedness. :D